modules/lm-coursier/src/main/scala/lmcoursier/internal/Resolvers.scala

```diff
@@ -52,6 +52,11 @@ object Resolvers {
         None
     }
 
+  // this handles whitespace in path
+  private def pathToUriString(path: String): String = {
+    "file://" + path.replaceAllLiterally(" ", "%20")
```

Comment:
Is it the best way to go from a file path to a URI? Wouldn't `new File(path).toURI.toASCIIString` work better? (It works fine around [here](https://github.com/coursier/coursier/blob/66b6b1781b86c63781b72302b32da5633aaae384/modules/coursier/jvm/src/main/scala/coursier/LocalRepositories.scala#L19) for example).
```suggestion
    new java.io.File(path).toURI.toASCIIString
```

Reply:
I tried that first but it won't work because `ivyPatterns` contain characters like parenthesis.

Reply:
Ah ok. Let's go with that then.